README.md

```diff
@@ -1,6 +1,6 @@
 # ipyannotator - the infinitely hackable annotation framework
 
-
+![CI-Badge](https://github.com/palaimon/ipyannotator/workflows/CI/badge.svg)
```

Comment:
![image](https://user-images.githubusercontent.com/1497140/102970818-804c6300-44f8-11eb-9df5-fd3b049ac296.png)

Badge doesn't seem to render properly on your branch. Will this change as soon as we merge? (path seems to point to palaimon repo)

Reply:
I think it will render after you setup org repo and ci workflow run happen